server.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// const fs = require('fs');
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+const cowsay = require('cowsay');
+const parseBody = require('./lib/parse-body.js');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer(function(req, res) {
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if(req.method === 'GET' && req.url.pathname === '/') {
+    res.writeHead(200,{'Content-Type': 'text/plain'});
+    res.end();
+  }
+
+  if(req.method === 'GET' && req.url.pathname === '/cowsay' && req.url.query.text) {
+    res.writeHead(200);
+    res.write(cowsay.say({text: req.url.query.text}));
+    res.end();
+  } else if (req.method === 'GET' && req.url.pathname === '/cowsay' && !req.url.query.text) {
```

Comment:
@kaylynyuh this can be refactored into a series of `if` statements and without the inclusion of `else` or `else if` statements - we'll talk about this more in the coming days but try using `return` instead of explicit `else` or `else if` statements on your next major piece of functionality that you build